realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -3034,8 +3034,8 @@ private void populateForDistinctInvalidTypesLinked(Realm realm) {
 
     @Test
     public void distinct() {
-        final long numberOfBlocks = 25;
-        final long numberOfObjects = 10; // Must be greater than 1
+        final long numberOfBlocks = 3;
```

Comment:
Performance change ?

Reply:
yups, creating rows takes lots of time in core when assertion enabled.